python/lsst/obs/decam/selectDecamImages.py

```diff
@@ -124,11 +131,19 @@ class SelectDecamImagesTask(BaseSelectImagesTask):
     def run(self, coordList, filter):
         """Select Decam images suitable for coaddition in a particular region
 
-        @param[in] filter: filter for images (one of g", "r", "i", "z", Y")
-        @param[in] coordList: list of coordinates defining region of interest
+        Parameters
+        ----------
+        filter :
+            filter for images (one of g", "r", "i", "z", Y")
+        coordList :
+            list of coordinates defining region of interest
+
+        Returns
+        -------
+        result : `struct`
+            return a pipeBase Struct containing:
```

Comment:
Reworking this: https://developer.lsst.io/python/numpydoc.html#struct-return-types